src/cloudWatchLogs/commands/selectLogStream.ts

```diff
@@ -0,0 +1,161 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as picker from '../../shared/ui/picker'
+import { MultiStepWizard, WizardStep } from '../../shared/wizards/multiStepWizard'
+import { LogGroupNode } from '../explorer/logGroupNode'
+import { CloudWatchLogs } from 'aws-sdk'
+import { ext } from '../../shared/extensionGlobals'
+import { CloudWatchLogsClient } from '../../shared/clients/cloudWatchLogsClient'
+import * as telemetry from '../../shared/telemetry/telemetry'
+
+export interface SelectLogStreamResponse {
+    region: string
+    logGroupName: string
+    logStreamName: string
+}
+
+export async function selectLogStream(node: LogGroupNode): Promise<void> {
+    let result: telemetry.Result = 'Succeeded'
+    const logStreamResponse = await new SelectLogStreamWizard(node).run()
+    if (logStreamResponse) {
+        vscode.window.showInformationMessage(
+            `Not implemented but here's the deets:
```

Comment:
localize? also lol

Reply:
No point in localizing, the next change will put some functionality here.